packages/editor/src/components/post-author/enhanced-post-author.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import { debounce } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import ComboboxControl from '../../../../components/build/combobox-control/';
```

Comment:
This can be imported from the components package.

Reply:
I tried and it didn't work, anything I need to do to have it recognized correctly? Do you need to merge the other PR first and publish the package?

Reply:
Any tips on how? When I try:

`import { ComboboxControl } from '@wordpress/components';`

I get an error at run time:
`React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.`

Reply:
The component is not exported yet. You can add it to the index of the components package.

Reply:
ok, i'll try to hunt that down - do you need to add that to your PR before merging?

Reply:
It was already merged.

Reply:
excellent, I saw that! I'll update the import and try again, should `import { ComboboxControl } from '@wordpress/components';`` work directly now or is there something else I need to do?

Reply:
got this working once i exported the component.